### PR TITLE
Update release.yaml

### PR DESCRIPTION
**Pull request by junkil-park**
### Description
reset `release.yaml`

### CODE REVIEW

**Review by thepomeranian — APPROVED** [review]

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 1]
```diff
@@ -1,66 +1,14 @@
 ---
 remote_endpoint: ~
-name: "v1.9"
+name: "v1.10"
 proposals:
-  - name: step_1_increase_max_txn_gas
```
Comment (igor-aptos) [inline aptos-move/aptos-release-builder/data/release.yaml:5]: we need to keep this step , until exception for framework execution max gas charging is implemented

**Review by igor-aptos — COMMENTED** [review]
should we be listing features that need to be enabled?

**Review by perryjrandall — APPROVED** [review]

**Review by igor-aptos — CHANGES_REQUESTED** [review]
step_1_increase_max_txn_gas needs to be there

File: aptos-move/aptos-release-builder/data/release.yaml (hunk 1 above)
Comment (junkil-park) [reply aptos-move/aptos-release-builder/data/release.yaml:5]: I added the step back. How does it look?

**Review by igor-aptos — APPROVED** [review]
looks good now

**Comment by junkil-park** [comment]
I locally tested the `release.yaml` for syntax check and proposal generation. We should do the `framework_upgrade` CI test when the test becomes available.

**Pull request merged**